docs/guides/intro-to-graphql.md

```diff
@@ -1,9 +1,407 @@
 <!--[meta]
-section: tutorials
+section: guides
 title: GraphQL
-draft: true
 [meta]-->
 
-# An Introduction To KeystoneJS's GraphQL API
+# Introduction to the GraphQL API
 
-In this tutorial we will introduce you to GraphQL and show you show to use the KeystoneJS GraphQL API to interact with your system.
+_Before you begin:_ This guide assumes you have running instance of Keystone with the GraphQL App configured, and a list with some data to query.
+
+Examples in this guide will refer to a `Users` list, however the queries, mutations and methods listed here would be the same for any Keystone list.
+
+For each List, Keystone generates four top level queries. Given the following example:
+
+```javascript
+keystone.createList('User', {
+  fields: {
+    name: { type: Text },
+  },
+});
+```
+
+## Queries
+
+Keystone would generate the following queries:
+
+- `allUsers`
+- `_allUsersMeta`
+- `User`
+- `_UserMeta`
+
+### allUsers
+
+Retrieves all items from the user list. The `allUsers` queries accept a set standard list of query arguments that allow you to search, limit and filter results. See: [Search and Filtering](#search-and-filtering).
+
+#### Usage
+
+```gql
+query {
+  allUsers {
+    id
+  }
+}
+```
+
+### \_allUsersMeta
+
+Retrieves all items meta for the user list. Currently used to return a count of all items which can be used for pagination. The `_allUsersMeta` query accepts the same [search and filtering](#search-and-filtering) parameters as the `allUsers` query.
+
+#### Usage
+
+```gql
+query {
+  _allUsersMeta {
+    count
+  }
+}
+```
+
+### User
+
+Retrieves a single item from the User list. The single entity query accepts a where parameter which must have provide an id.
+
+#### Usage
+
+```gql
+query {
+  User(where: { id: ID }) {
+    name
+  }
+}
+```
+
+### \_UserMeta
+
+Unlike `_allUsersMeta` this returns metadata relating to the list such as access control information. This query accepts no parameters.
+
+## Mutations
+
+For each list Keystone generates six top level mutations:
+
+- `createUser`
+- `createUsers`
+- `updateUser`
+- `updateUsers`
+- `deleteUser`
+- `deleteUsers`
+
+### createUser
+
+Add a single User to the Users list. Requires a `data` parameter that is an object where keys match the field names in the list definition and the values are the data to create.
```

Comment:
There's a minor inconsistency in this page; the list is sometimes referred to as a mix of upper case and lower case and singular and plural. I think it best to settle on the list name in the singular form, wrapped in backticks to ensure it is obvious we're referring what the list is called. ie; `User`.

Reply:
Sure!